CMakeLists.txt

```diff
@@ -29,7 +29,9 @@ cc_set_if_undefined(USE_PHYSICS_PHYSX        OFF)
 cc_set_if_undefined(USE_MODULES              OFF)
 
 
-if(ANDROID OR WINDOWS)
+if(NINTENDO)
+    cc_set_if_undefined(CC_USE_NVN ON)
```

Comment:
`CC_USE_NX` or `CC_USE_NINTENDOSDK`

Reply:
Here we need the name of the graphics backend of NX, not NX itselff. I don't think `USE_NX` makes any sense here